src/components/LoginModule/SignInPassword/SignInPassword.js

```diff
@@ -3,6 +3,10 @@ import { useEffect, useState } from "react";
 import { Col, Container, Row } from "react-bootstrap";
 import { isAuthenticated } from "../../../core/common.functions";
 import { AppRoutes } from "../../../core/constant";
+import { PrimaryButton } from "../../../shared/buttons/PrimaryButton";
```

Comment:
Put this all in to index.js and do export once { list of import } index.js

Reply:
Done for icons, deleted buttons folder and updated custom.scss